tests/python/pytorch/graphbolt/test_feature_store.py

```diff
@@ -1,13 +1,28 @@
+import numpy as np
 import pytest
 import torch
 from dgl import graphbolt as gb
 
 
-def test_in_memory_feature_store():
+def to_on_disk_tensor(name, t):
```

Comment:
Other than unittest, we should also test the performance and add E2E tests benchmark for canonical solutions in the regression framework #5916.

Reply:
Is there any instructions to write the benchmark?